### ISSUE TITLE
Square brackets in comments with paredit are not behaving correctly

### ISSUE THREAD

**Issue by noncom** [laurentpetit/ccw#843]
I have noticed that when paredit is on and I type `[` in a comment it becomes `[|]`. Then, if I want to leave it like that and move on and I type `]`, it becomes `[]|]` instead of expected `[]|` or even `[] |`. This behavior differs from the behavior in code and is not really appropriate (imho).

**Comment by laurentpetit** [comment]
Indeed. Question is: what should the behavior be in your mind? Same as in
code, or no addition of closing paren at all? The latter is certainly
easier to get at from an implementation point of view.

Le dimanche 23 août 2015, noncom <notifications@github.com> a écrit :

> I have noticed that when paredit is on and I type [ in a comment it
> becomes [|]. Then, if I want to leave it like that and move on and I type
> ], it becomes []|] instead of expected []| or even [] |. This behavior
> differs from the behavior in code and is not really appropriate (imho).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laurentpetit/ccw/issues/843>.
>


-- 
Laurent Petit

**Comment by noncom** [comment]
I think same as in code is more appropriate since that's the nature of how `[` and `]` are used even in comments. However, if this is not straightforward to implement, then the simple solution to prevent the addition of the closing pair is ok since this already fixes the primary bug and removes the distraction that it provides to the user.
